thirdparty/CMakeLists.txt

```diff
@@ -20,19 +20,65 @@ endfunction()
 
 include(CMakeDependentOption)
 
+set(ADIOS2_USE_EXTERNAL_DEPENDENCIES OFF CACHE BOOL
+  "Use externally supplied dependencies")
+set(ADIOS2_USE_EXTERNAL_GTKORVO_STACK "${ADIOS2_USE_EXTERNAL_DEPENDENCIES}" CACHE BOOL
```

Comment:
AFAICT EVPath is the only actual dependency we're using and all the other pieces, i.e. atl, ffs, dill, enet, are dependencies of EVPath.  @eisenhauer is that right?

If that's the case then what does `ADIOS2_USE_EXTERNAL_GTKORVO_STACK` give you that `ADIOS2_USE_EXTERNAL_EVPATH` doesn't?

Reply:
It controls the default value for `ADIOS2_USE_EXTERNAL_<gtkorvo-dependency>`. It is possible, though admittedly unlikely, to use the built-in EVPath while using an external atl/ffs/etc.

I suppose I could take this option out and just make the default `OFF` for all of them. Then anyone who wants to use an external version has to explicitly set it to `ON`, which most likely just means setting `ADIOS2_USE_EXTERNAL_EVPATH` to `ON`.

Reply:
Isn't it already setup that way but with `cmake_dependent_option`? The only thing I see this really controlling is a shortcut to using an internal EVPath with external dependencies, which is really a one off use case that seldom, if ever, people would use.  And even if they did they can still add the extra options for it.